src/basis/basis.js

```diff
@@ -396,18 +413,41 @@ export class Basis {
      */
     get maxPairwiseDistance() {
         let maxDistance = 0;
-        for (let i = 0; i < this._elements.array.length; i++) {
-            for (let j = i + 1; j < this._elements.array.length; j++) {
-                const distance = math.vDist(
-                    this._coordinates.getArrayElementByIndex(i), this._coordinates.getArrayElementByIndex(j));
-                if (distance > maxDistance) {
-                    maxDistance = distance;
+        if (this._elements.array.length >= 2) {
+            for (let i = 0; i < this._elements.array.length; i++) {
+                for (let j = i + 1; j < this._elements.array.length; j++) {
+                    const distance = math.vDist(
+                        this._coordinates.getArrayElementByIndex(i), this._coordinates.getArrayElementByIndex(j));
+                    if (distance > maxDistance) {
+                        maxDistance = distance;
+                    }
                 }
             }
         }
+        console.log(maxDistance);
```

Comment:
Will remove on next set of revisions.

Reply:
We might as well just clean it ahead of time or not commit at all ...